src/algorithms/masked_gibbs.jl

```diff
@@ -510,3 +510,50 @@ function assert_spikes_not_in_mask(
         end
     end
 end
+
+function clean_masks(
+    masks::Vector{Mask},
+    num_neurons::Integer
+)
+    new_masks = Mask[]
+    for n = 1:num_neurons
+        # Find the masks corresponding to thie neuron
+        this_neur_masks = Mask[]
+        for mask = 1:length(masks)
+            if masks[mask][1] == n
+                push!(this_neur_masks, masks[mask])
+            end
+        end
+
+        # Find the start of mask times for this neuron
+        start_times = []
+        for (n,(t0,t1)) in this_neur_masks
+            push!(start_times, t0)
+        end
+
+        # Sort them
+        start_times_indices = sortperm(start_times)
+        sorted_mask = this_neur_masks[start_times_indices]
```

Comment:
Since `start_times_indices` isn't re-used we could reduce this down to one line. Also amend the comment to be more informative.

```julia
# Sort masks by start time.
sorted_mask = this_neur_masks[sortperm(start_times)]
```